deploy/osd-must-gather-operator/01-osd-devaccess.Role.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: osd-devaccess
+  namespace: openshift-must-gather-operator
+rules:
+- apiGroups:
+  - ""
```

Comment:
Do they really need to be able to create **all** kinds of secrets?

Reply:
no, I suppose not :)